torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -638,16 +652,211 @@ c10::optional<std::vector<int64_t>> GetValueFromListConstructNode(
       : c10::nullopt;
 }
 
+void ProcessBroadCastNode(Node* n) {
+  if (ConstantValueMap::HasShape(n->input(0)->debugName()) &&
```

Comment:
Maybe add an assertion initially that n has 2 inputs.

Reply:
done